.github/workflows/builds.yml

```diff
@@ -297,10 +297,9 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.FERDIUM_PUBLISH_TOKEN }}
           CSC_IDENTITY_AUTO_DISCOVERY: false
-          SNAPCRAFT_LOGIN: ${{ secrets.SNAPCRAFT_LOGIN }}
+          SNAPCRAFT_STORE_CREDENTIALS: ${{ secrets.SNAPCRAFT_LOGIN }}
         run: |
-          sudo snap install snapcraft --channel=5.x/stable --classic
-          echo "$SNAPCRAFT_LOGIN" | snapcraft login --with -
+          sudo snap install snapcraft --channel=7.x/stable --classic
```

Comment:
Should a comment be left here explaining that login is no longer required?

Reply:
Yes, that would be a good idea, by adding a link to the comment on the snapcraft forum that details what the documentation doesn't.